umm-spec-lib/test/cmr/umm_spec/test/migration/version/collection.clj

```diff
@@ -3838,85 +3838,192 @@
+                                 :Name    "UMM-C",
+                                 :Version "1.18.1"}}
+        ;; sample-collection
+        {:AssociatedDOIs        [{:Type "IsPreviousVersionOf"}
+                                 {:Type "IsNewVersionOf"}
+                                 {:Type "Field Campaign"}]
+         :DOI                   {:DOI "10.5067/fake.record.01"}
+         :SpatialExtent         {:GranuleSpatialRepresentation "NO_SPATIAL"}
+         :TemporalExtents       [{:EndsAtPresentFlag false
+                                  :SingleDateTimes   ["2024-02-14T01:02:03Z"]}]
+         :MetadataSpecification {:URL     "https://cdn.earthdata.nasa.gov/umm/collection/v1.18.2",
+                                 :Name    "UMM-C",
+                                 :Version "1.18.2"}}
+
+        "Migrating CollectionProgress enum PREPRINT back to PLANNED"
+        ;; expected
+        {:CollectionProgress    "PLANNED"
+         :MetadataSpecification {:URL     "https://cdn.earthdata.nasa.gov/umm/collection/v1.18.1",
+                                 :Name    "UMM-C",
+                                 :Version "1.18.1"}}
+        ;; sample-collection
+        {:CollectionProgress    "PREPRINT"
+         :MetadataSpecification {:URL     "https://cdn.earthdata.nasa.gov/umm/collection/v1.18.2",
+                                 :Name    "UMM-C",
+                                 :Version "1.18.2"}}
+
+        "Migrating CollectionProgress enum PREPRINT back to PLANNED"
+        ;; expected
+        {:CollectionProgress    "PLANNED"
+         :MetadataSpecification {:URL     "https://cdn.earthdata.nasa.gov/umm/collection/v1.18.1",
+                                 :Name    "UMM-C",
+                                 :Version "1.18.1"}}
+        ;; sample-collection
+        {:CollectionProgress    "INREVIEW"
+         :MetadataSpecification {:URL     "https://cdn.earthdata.nasa.gov/umm/collection/v1.18.2",
+                                 :Name    "UMM-C",
+                                 :Version "1.18.2"}}
+
+        "Migrating CollectionProgress enum SUPERSEDED back to COMPLETE"
+        ;; expected
+        {:CollectionProgress    "COMPLETE"
+         :MetadataSpecification {:URL     "https://cdn.earthdata.nasa.gov/umm/collection/v1.18.1",
+                                 :Name    "UMM-C",
+                                 :Version "1.18.1"}}
+        ;; sample-collection
+        {:CollectionProgress    "SUPERSEDED"
+         :MetadataSpecification {:URL     "https://cdn.earthdata.nasa.gov/umm/collection/v1.18.2",
+                                 :Name    "UMM-C",
+                                 :Version "1.18.2"}}))
+
+(def sample-1-18-3-record
+  {:RelatedUrls [{:Description "Contact group related url description"
+                  :URLContentType "DistributionURL"
+                  :Type "GET DATA"
+                  :Subtype "ECHO"
+                  :URL "www.contact.group.foo.com"
```

Comment:
FYI foo.com is a real site, example.org is better.

Reply:
Since this example doesn't go anywhere or the URL isn't parsed, I am just going to leave it.